src/python/pants/notes/1.15.x.rst

```diff
@@ -3,6 +3,41 @@
 This document describes releases leading up to the ``1.15.x`` ``stable`` series.
 
 
+1.15.0rc1 (3/25/2019)
+---------------------
+
```

Comment:
Should we add a note that this release will not work with Py37 wheel due to a temporary bug? 

```
Note this release does not support using Python 3.7 due to a bug we introduced to release a Python 3 PEX [add issue]. We are prioritizing fixing this before 1.15.0 is cut. In the meantime, you may use Python 2.7 or Python 3.6.
```

We could alternatively include this in the email, but maybe there's value to being in the published docs?

Reply:
Do you mean pants built as a pex doesn't work under 3.7? Pants run via the setup script just tested out ok for me. I'm pretty sure most users use the setup script. So a note is warranted but the language is much too strong as currently suggested.

Reply:
No, I mean Pants downloaded as a wheel will fail to resolve, particularly if you try to use any contrib packages. In the setup repo, apply the below diff then run `./pants`.

```diff
diff --git a/pants.ini b/pants.ini
index 554f2ab..7800964 100644
--- a/pants.ini
+++ b/pants.ini
@@ -1,6 +1,6 @@
 [GLOBAL]
-pants_version : 1.15.0.dev4
-pants_runtime_python_version : 3.6
+pants_version : 1.15.0rc0
+pants_runtime_python_version : 3.7
 plugins : [
        'pantsbuild.pants.contrib.go==%(pants_version)s',
        ]
```

You'll get this error message:
```
INFO] Resolving new plugins...:
  pantsbuild.pants.contrib.go==1.15.0rc0
timestamp: 2019-03-25T13:12:49.519219
Exception caught: (pex.resolver.Unsatisfiable) (backtrace omitted)
Exception message: Could not satisfy all requirements for pantsbuild.pants==1.15.0rc0:
    pantsbuild.pants==1.15.0rc0(from: pantsbuild.pants.contrib.go==1.15.0rc0)
```

If you change the version back to `1.15.0.dev4`, before we made the change to pinning cryptography, then it all works.

Alternatively, if you remove the contrib entry, it works.

--

Perhaps reword to this:
```
Note if you use any contrib packages, this release has a bug that does not allow running Pants with Python 3.7; you must instead temporarily use Python 2.7 or Python 3.6 to use contrib packages. We are prioritizing fixing this before 1.15.0 is cut.
```

Reply:
Gotcha. This is me coming up to speed, but the issues are:
1. pex can't resolve abi3 things correctly
2. pants is a bad actor in that you can't build a wheel from a source release (we don't even release an sdist).

And since pants uses the pex machinery to resolve plugins the bug of 1 leads to the bug of 2 leads to no solution.